### PR TITLE
Add template.md

### PR DESCRIPTION
**Pull request by guitassinari**
For better viewing:
[0000-template.md](https://github.com/Enactus-UFRGS/enactus-team-management-rfcs/blob/695d5f866c42d4b6e7b8a8e2931684d86d18792e/0000-template.md)

PS: this a first PR and it is being used to teach new developers how the code review flow works.

### CODE REVIEW

**Comment by LigiaBatista** [comment]
Nice!!

**Comment by daviolrb** [comment]
So, let's do it

**Comment by MelissaFil** [comment]
Desculpa a demora. É pra comentar em ingres ? rs

File: 0000-template.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+
+Brief explanation of the feature.
+
+# Basic example
```
Comment (MelissaFil) [inline 0000-template.md:9]: A primeira que eu omito rs

**Review by daviolrb — APPROVED** [review]
So far so good 💃

**Review by daviolrb — APPROVED** [review]
Here we go

**Review by LigiaBatista — CHANGES_REQUESTED** [review]
improve it, please!!

**Review by LigiaBatista — CHANGES_REQUESTED** [review]
improve it !!

**Pull request merged**